### PR TITLE
ome-files 0.5.0: Fix download links and sizes

### PR DESCRIPTION
**Pull request by rleigh-codelibre**
Fix links for 0.5.0 release

### CODE REVIEW

**Comment by snoopycrimecop** [comment]
Empty PR description. Please add a short summary of the PR scope and some testing instructions.

File: ome-files/downloads/index.html [hunk 1]
```diff
@@ -29,8 +29,8 @@ <h5>CentOS 7.3</h5>
                     <div class="card-section">
                         <img alt="CentOS logo" src="{{ site.baseurl }}/img/logos/3rd-party_centos.png">
                         <h5>CentOS 6.8</h5>
```
Comment (hflynn) [inline ome-files/downloads/index.html]: You need to update this header too

File: ome-files/downloads/index.html [hunk 2]
```diff
@@ -49,8 +49,8 @@ <h5>MacOS X 10.12</h5>
                     <div class="card-section">
                         <img alt="FreeBSD logo" src="{{ site.baseurl }}/img/logos/3rd-party_freebsd.png">
                         <h5>FreeBSD 11.0</h5>
```
Comment (hflynn) [inline ome-files/downloads/index.html]: again header needs updated too

**Comment by hflynn** [comment]
Two headers to fix

File: ome-files/downloads/index.html (hunk 1 above)
Comment (rleigh-codelibre) [reply ome-files/downloads/index.html]: Done

File: ome-files/downloads/index.html (hunk 2 above)
Comment (rleigh-codelibre) [reply ome-files/downloads/index.html]: Done

**Comment by hflynn** [comment]
👍 links all work now

**Comment by joshmoore** [comment]
Thanks all.

**Pull request merged**